src/Sylius/Component/Promotion/Model/CatalogPromotionInterface.php

```diff
@@ -21,6 +21,10 @@
 
 interface CatalogPromotionInterface extends ResourceInterface, CodeAwareInterface, TranslatableInterface
 {
+    public const TYPE_PERCENTAGE_DISCOUNT = 'percentage_product_discount';
```

Comment:
Why did you change the place of these costs? shouldn't we have types of rules on the rule Interface and types of actions on the Action interface?

Reply:
it wasnt needed :D